config.go

```diff
@@ -6,17 +6,25 @@ import (
 	"os"
 )
 
+type GoogleConfig struct {
+	Project string
+	LogName string
+}
+
 type BaseConfig struct {
 	ServerType ServerType
 	ConfigName string
 	Version    string
+	Google     GoogleConfig
 }
 
 type IBaseConfig interface {
 	GetVersionString() string
 	SetServerType(string)
 	GetServerType() ServerType
 	IsProductionServer() bool
+	GetGoogleProject() string
```

Comment:
Are these getters useful or necessary, as opposed to just accessing the Google property of BaseConfig directly?

Reply:
Ah yeah they're not necessary. I think I originally had the embedded GoogleConfig implementing an interface and left them in. I'll remove them.